wodles/gcloud/buckets/bucket.py

```diff
@@ -159,6 +159,28 @@ def _update_last_processed_files(self, processed_files: list[storage.blob]):
                                                                                 creation_time=blob.time_created))
     processed_files = list()
 
+    def _get_last_creation_time(self):
+        """Get the latest creation time value stored in the database for the given project, bucket_name and
+        prefix.
+
+        Returns
+        -------
+        datetime or None
+            The datetime of the last log parsed or None if no log have been parsed yet for that bucket.
+        """
+        creation_time = None
```

Comment:
```suggestion
        creation_time = datetime.min.replace(tzinfo=pytz.UTC)
```
If you set the default creation_time to this value you will avoid issues derived from having to handle `None` values returned by this function, simplifying the `process_data` function.